From be9bb7163c2a29cd8e5ddea24226ea059a6ed224 Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Fri, 27 Sep 2019 16:20:49 +0200
Subject: [PATCH] Integrate machine independent resin environment configuration for U-Boot 2022.01

This patch integrates resin default environment configuration in u-boot 2022.01
which has Kconfig support. Updated to work with the newer env_default.h structure.

Signed-off-by: Andrei Gherzan <andrei@resin.io>
Signed-off-by: Vicentiu Galanopulo <vicentiu@balena.io>

Upstream-Status: Inappropriate [configuration]
---
 include/env_default.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index 86b639d3e2..b15bb7eed5 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -7,6 +7,8 @@
  * Wolfgang Denk, DENX Software Engineering, wd@denx.de.
  */
 
+#include <env_resin.h>
+
 #include <env_callback.h>
 
 #ifdef DEFAULT_ENV_INSTANCE_EMBEDDED
@@ -21,6 +23,7 @@ static char default_environment[] = {
 #else
 const uchar default_environment[] = {
 #endif
+	BALENA_ENV
 #ifndef CONFIG_USE_DEFAULT_ENV_FILE
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
-- 
2.17.1
